SofaKernel/framework/sofa/core/Mapping.inl

```diff
@@ -127,7 +127,7 @@ sofa::defaulttype::BaseMatrix* Mapping<In,Out>::createMappedMatrix(const behavio
     sofa::defaulttype::BaseMatrix* result;
     if( !this->areMatricesMapped() )
     {
-        sout << "Mapping::createMappedMatrix() this mapping do not support matrices building. Set mapMatrices to true" << getClassName() << sendl;
+        msg_warning() << "Mapping::createMappedMatrix() this mapping do not support matrices building. Set mapMatrices to true" << getClassName();
```

Comment:
To be super pinchy (merge it witjout taking that into account)...
There is no need to add "Mapping::createMappedMatrix()" to the message as the emitting point is already in the msg_warning() macro. (additionally it make no sense to send to the user any details about c++ stuff)